docs/pages/OneFilter/OF-Levels.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Levels
+parent: OneFilter User Guide
+nav_order: 8
+---
+
+# OneFilter
+{: .no_toc }
+Quickly find elements between Levels and Reference Planes.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+# Levels
+
+The Levels tab allows the user to select categories of elements to find between Levels and Reference Plane.
+
+## Select Category
+
+1. The first step is to select whether you want to apply filters based on the Whole Model or the Active View. Select the radio button to choose the view.
+
+![OneFilter select model view](../../../assets\images\OneFilter\OF-Lv-WholeModel.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+2. Now, use the checkboxes to select the categories to filter. Note that, once you have selected the categories, on the right table will show the existing families, with the category, family name, type name, Level and count columns.
```

Comment:
- Capitalize "Categories"
- "Note that, once you have selected the Categories, ~~on~~ the right table will show the existing Families, with its Category, Family name, Type name, Level and count columns."